docs/developers/Custom_Rules.mdx

```diff
@@ -233,15 +233,17 @@ In that case, it is best to add an explicit type declaration.
 This rule snippet targets name nodes of both function and interface declarations:
 
 ```ts
-import { AST_NODE_TYPES, ESLintUtils } from '@typescript-eslint/utils';
+import {
+  AST_NODE_TYPES,
+  ESLintUtils,
+  TSESTree,
+} from '@typescript-eslint/utils';
```

Comment:
It looks like `AST_NODE_TYPES` is no longer used in this example. What do you think about removing it?

```suggestion
import { TSESTree, ESLintUtils } from '@typescript-eslint/utils';
```

Reply:
sure. `ESLintUtils` is also not used in this snipped, removed.